src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py

```diff
@@ -82,8 +85,7 @@ def _get_node(self, **all_args):
 
         # validate output
         if output_columns is None:
-            raise ValueError(
-                "'None' output passed when it cannot be none.")
```

Comment:
this one is needed. Pls see example how to use this transform, it should be clear from it

Reply:
The change has been reverted to the previous code in commit 8afb27968c6eb9e1c3714561dedb64fb9e34e672.

---
In reply to: [347187028](https://github.com/microsoft/NimbusML/pull/364#discussion_r347187028) [](ancestors = 347187028)